unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/AvatarVisibilityTests.cs

```diff
@@ -0,0 +1,56 @@
+using AvatarShape_Tests;
+using DCL;
+using DCL.Helpers;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Tests
+{
+    public class AvatarVisibilityTests
```

Comment:
We are starting to use the `Should` naming scheme for tests, as it leads for shorter test names that are a bit more clear.

```suggestion
    public class AvatarVisibilityShould
```